From 5ac6a2eecfcb495a085bf838a9877d1e77d20bc0 Mon Sep 17 00:00:00 2001
From: Piyush Raj Chouhan <103498040+Pc1598@users.noreply.github.com>
Date: Sun, 9 Nov 2025 09:33:25 +0000
Subject: [PATCH] Input: xpad - add support for Cosmic Byte Blitz Gamepad

---
 drivers/input/joystick/xpad.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/input/joystick/xpad.c b/drivers/input/joystick/xpad.c
index d72e89c25e5031..66fda6f71a1e85 100644
--- a/drivers/input/joystick/xpad.c
+++ b/drivers/input/joystick/xpad.c
@@ -390,6 +390,8 @@ static const struct xpad_device {
 	{ 0x24c6, 0x5b03, "Thrustmaster Ferrari 458 Racing Wheel", 0, XTYPE_XBOX360 },
 	{ 0x24c6, 0x5d04, "Razer Sabertooth", 0, XTYPE_XBOX360 },
 	{ 0x24c6, 0xfafe, "Rock Candy Gamepad for Xbox 360", 0, XTYPE_XBOX360 },
+	{ 0x2563, 0x028e, "Cosmic Byte Blitz Wired Gamepad", 0, XTYPE_XBOX360 },
+	{ 0x2563, 0x0575, "Cosmic Byte Blitz Wireless Gamepad", 0, XTYPE_XBOX360 },
 	{ 0x2563, 0x058d, "OneXPlayer Gamepad", 0, XTYPE_XBOX360 },
 	{ 0x294b, 0x3303, "Snakebyte GAMEPAD BASE X", 0, XTYPE_XBOXONE },
 	{ 0x294b, 0x3404, "Snakebyte GAMEPAD RGB X", 0, XTYPE_XBOXONE },
